src/main/java/com/hcl/appscan/jenkins/plugin/builders/AppScanBuildStep.java

```diff
@@ -9,6 +9,7 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.Serializable;
+import java.net.Proxy;
```

Comment:
why is this import required? there is no changes related to Proxy

Reply:
Removed this import.